src/components/CreateForm/index.js

```diff
@@ -12,25 +12,74 @@ export default function CreateForm() {
 	const handleSubmit = event => {
 		event.preventDefault();
 		addMeal(newMeal);
-		setNewMeal({title: ''});
+		setNewMeal({
+			title: '',
+			ingredients: '',
+			preparation: '',
+			servings: '',
+			time: '',
+		});
 	};
 
 	return (
 		<>
 			<StyledForm onSubmit={handleSubmit}>
 				<h2>Plan your next meal!</h2>
-				<fieldset>
+				<fieldset className="inputfieldset">
```

Comment:
Why do you use classnames at this point? It would be better to create a new styled component instead of a class.

Reply:
Thanks for your feedback!
I will implement it and post again.